java/blade/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>0.0.1</version>
+
+    <properties>
+        <blade-mvc.version>2.0.15.RELEASE</blade-mvc.version>
+    </properties>
+
+    <dependencies>
+        <!-- mvc dependency -->
+        <dependency>
+            <groupId>com.bladejava</groupId>
+            <artifactId>blade-mvc</artifactId>
+            <version>${blade-mvc.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.3.0</version>
```

Comment:
need to keep version of plugins ?

PS : I don't know if it is safe to keep (or remove) this

Reply:
I feel safe using the latest. It doesn't change that often.

Reply:
What is considered as `safe`, honestly I don't know ... (since I know only for `ruby` or `python`).
However, the stand point `here` is to have the same behavior (plugin version) for all `java` **frameworks**, I think it's better to keep the same versions for a fair (at least not unfair) comparison
wdyt ?

Reply:
Ok.